docs/faq.rst

```diff
@@ -406,3 +406,16 @@ Example: ::
         await ctx.send(f'Pushing to {remote} {branch}')
 
 This could then be used as ``?git push origin master``.
+
+Application Commands
+--------------------
+
+Questions regarding Discord's new Application Commands.
```

Comment:
```suggestion
Questions regarding Discord's new application commands, frequently known as "slash commands" or "context menu commands".
```

Reply:
Slash commands and context menu commands are not alternate names for application commands, rather they are types of application commands.

Reply:
I don't see the relevance for the note. Most users will call them slash commands or context menu commands, that's what that note is saying. The API calls them application commands, yes, but users and developers just call them something else.

Reply:
The problem is that **slash commands** and **context menu commands** _are not the same thing_.

Reply:
Sure. Except this has nothing to do with the note which is there to help users understand what an application command is from their perspective. This type of pedantry is unnecessary and is just holding back the merging of this PR.

Reply:
That's a valid point, but I was worried about someone calling context menu commands "slash commands" or vice versa.